datahub-web-react/src/app/entity/user/UserOwnership.tsx

```diff
@@ -35,7 +35,15 @@ export default ({ ownerships, entityPath }: Props) => {
                     </Typography.Title>
                 }
                 renderItem={(item) => {
-                    return entityRegistry.renderPreview(entityType, PreviewType.PREVIEW, item);
+                    return (
+                        <div
+                            style={{
```

Comment:
Nit: can we extract out styles into a `const` variable at the top of the file?

Reply:
Removed inline style